src/global/global_init.cc

```diff
@@ -261,6 +261,11 @@ int global_init_prefork(CephContext *cct, int flags)
 {
   if (g_code_env != CODE_ENVIRONMENT_DAEMON)
     return -1;
+    
+  if (test_pid_file_in_use(g_conf) < 0) {
```

Comment:
```c
fd = ::open(pid_file, O_CREAT|O_WRONLY, 0644));
```
if "pid_file" is an empty string, the `open()` call fails, and `exit(1)` is called. the process terminates. which is unexpected.